packages/web-runtime/src/index.ts

```diff
@@ -216,10 +216,6 @@ export const bootstrapApp = async (configurationPath: string): Promise<void> =>
       await spacesStore.loadSpaces({ graphClient: clientService.graphAuthenticated })
       const personalSpace = spacesStore.spaces.find(isPersonalSpaceResource)
 
-      if (!personalSpace) {
-        return
-      }
-
```

Comment:
The idea was to have the next statement inside the block:

```
if (personalSpace) {
  spacesStore.updateSpaceField(...)
}
```

Reply:
got it, updated